socs/agents/stm_thermo/agent.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+'''OCS agent for thermometer board in the stimulator box
+'''
+import argparse
+import os
+import time
+
+import txaio
+from ocs import ocs_agent, site_config
+from ocs.ocs_twisted import TimeoutLock
+
+from socs.agents.stm_thermo.drivers import Max31856, from_spi_node_path
+
+LOCK_RELEASE_SEC = 1.
+LOCK_RELEASE_TIMEOUT = 10
+
+
+class StmThermometerAgent:
```

Comment:
Forgot to add to my review: could you change to `StimThermometerAgent` and change elsewhere when needed?